lib/formtastic-bootstrap/form_builder.rb

```diff
@@ -1,3 +1,6 @@
+require 'formtastic-bootstrap/inputs'
+require 'formtastic-bootstrap/actions'
+
```

Comment:
Instead of requiring these here, either the autoloader needs to be fixed here: https://github.com/mjbellantoni/formtastic-bootstrap/blob/master/lib/formtastic-bootstrap.rb
Or just always require these files and zap the autoloader.